### PR TITLE
Fix merge conflicts

### PR DESCRIPTION
**Pull request by sverholen**
- Reorder properties in pom to keep everything readable

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -25,20 +29,22 @@
 		<titanium-json-ld.version>1.3.0</titanium-json-ld.version>
 		<apache-jena.version>4.4.0</apache-jena.version>
 
-		<jupiter.version>5.8.2</jupiter.version>
 		<byte-buddy.version>1.12.10</byte-buddy.version>
-		<skipFormatCode>false</skipFormatCode>
```
Comment (WLefever-Cegeka) [inline pom.xml:30]: skipFormatCode is being used so you can't just remove it.

File: pom.xml [hunk 2]
```diff
@@ -226,7 +238,7 @@
 				<plugin>
 					<groupId>org.apache.maven.plugins</groupId>
 					<artifactId>maven-surefire-plugin</artifactId>
-					<version>${surefire.version}</version>
+					<version>${maven-surefire-plugin.version}</version>
```
Comment (WLefever-Cegeka) [inline pom.xml:241]: in the properties this is still called surefire.version.

**Review by WLefever-Cegeka — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (sverholen) [reply pom.xml:30]: It's on line 21, separated from properties for test dependencies, to keep the BOM manageable.

File: pom.xml (hunk 2 above)
Comment (sverholen) [reply pom.xml:241]: We're both right, line 49 needs to go.

**Pull request merged**